app/views/internal/users/_activity.erb

```diff
@@ -2,10 +2,10 @@
   <div class="col-sm-6">
     <h2>Activity</h2>
     <ul>
-      <li><%= @user.comments.size %> comments</li>
-      <li><%= @user.articles.size %> articles</li>
-      <li><%= @user.reactions.size %> reactions</li>
-      <li><%= @user.followers.size %> followers</li>
+      <li><%= @user.comments_count %> comments</li>
+      <li><%= @user.articles_count %> articles</li>
+      <li><%= @user.reactions_count %> reactions</li>
+      <li><%= @user.followers_count %> followers</li>
```

Comment:
Unless a specific user becomes really popular really quickly, I not sure if this will be a issue. But, I think we should ask the ones that really uses that page. WDYT?

Reply:
Hmm I think we mostly use these counts to check whether the user has any activity. Agree with your point about a popular user, so this change works. 👍